craco.config.js

```diff
@@ -19,26 +22,61 @@ const cracoConfig = (module.exports = {
       }
 
       // add rules to pack WASM (for Sourceror)
-      // adapted from https://prestonrichey.com/blog/react-rust-wasm/
       const wasmExtensionRegExp = /\.wasm$/;
       webpackConfig.resolve.extensions.push('.wasm');
       webpackConfig.module.rules.forEach(rule => {
         (rule.oneOf || []).forEach(oneOf => {
-          if (oneOf.loader && oneOf.loader.indexOf('file-loader') >= 0) {
-            // Make file-loader ignore WASM files
+          if (oneOf.type === 'asset/resource') {
             oneOf.exclude.push(wasmExtensionRegExp);
           }
         });
       });
+      // See https://webpack.js.org/configuration/experiments/#experiments.
+      webpackConfig.experiments = {
+        syncWebAssembly: true
+      };
       webpackConfig.output.webassemblyModuleFilename = 'static/[hash].module.wasm';
 
+      // Polyfill Node.js core modules.
+      // An empty implementation (false) is provided when there is no browser equivalent.
+      webpackConfig.resolve.fallback = {
+        'child_process': false,
+        'constants': require.resolve('constants-browserify'),
+        'fs': false,
+        'http': require.resolve('stream-http'),
+        'https': require.resolve('https-browserify'),
+        'os': require.resolve('os-browserify/browser'),
+        'stream': require.resolve('stream-browserify'),
+        'timers': require.resolve('timers-browserify'),
+        'url': require.resolve('url/')
```

Comment:
The trailing slash was part of the instructions given by Webpack. I'm not very sure if removing it will cause any errors.